utilities/build.sh

```diff
@@ -11,6 +12,15 @@ ACTIVE_TYPE=Main
 BUILD_TYPE=Debug
 BUILD_PLATFORM=Device
 
+while getopts 'c' flag; do
```

Comment:
The getopts is an internal flag parser. This will add flexibility for parameter ordering and optionality.

For example, the parameters of the script are optional `[main|initial] [dev|debug|release] [device|simulator]` but the following execution is wrong.
```
./build.sh release simulator
```

So using `getopts` will actually make the optional arguments.